experiment_scheduler/db_util/connection.py

```diff
@@ -0,0 +1,26 @@
+"""
+This file exists to manage sqlalchemy function
+"""
+import ast
+
+from sqlalchemy import create_engine, MetaData
+from sqlalchemy.orm import declarative_base, sessionmaker
+from sqlalchemy_utils import database_exists, create_database
+from experiment_scheduler.common.settings import USER_CONFIG
+
+db_url = ast.literal_eval(USER_CONFIG.get("default", "db_url"))

```

Comment:
Is it right to get `"db_url"` if there is no `default` key?

Reply:
@sinunu `(USER_CONFIG.get("default", "db_url"))` does not set "db_url" as default value.
It imports value of "db_url" in default setting
```python
[default]
db_url = "db_url"
```